pkg/actor/partitioned_update_test.go

```diff
@@ -40,3 +42,30 @@ func TestDeployedInCluster(t *testing.T) {
 		t.Fatal("we should find the file")
 	}
 }
+
+func TestGetImageNameNoVersion(t *testing.T) {
+	type testCase struct {
+		Name              string
+		ImageName         string
+		ExpectedImageName string
+	}
+	tests := []testCase{
+		{
+			Name:              "should_keep_the_sha256_format_complete",
+			ImageName:         "cockroachdb/cockroach@sha256:a1ef571ff3b47b395084d2f29abbc7706be36a826a618a794697d90a03615ada",
+			ExpectedImageName: "cockroachdb/cockroach@sha256:a1ef571ff3b47b395084d2f29abbc7706be36a826a618a794697d90a03615ada",
+		},
+		{
+			Name:              "should_remove_the_version_for_non_sha256_format_of_the_image",
+			ImageName:         "cockroachdb/cockroach:v20.2.0",
```

Comment:
I don't think this does what we want.  Customers should be able to use SHAs OR tags - a customer is probably going to manually set a tag, and the cockroachDBVersion API is always going to tag a SHA.  Unless I'm missing something here?

Reply:
@Keith I will rephrase... the sha256 format was for the RELATED_IMAGE that we add in the manifest, and which Openshift CI replaces them with sha256.

Reply:
okay maybe I misunderstood what the test was doing then